UefiPayloadPkg/Library/PlatformHookLib/PlatformHookLib.c

```diff
@@ -89,11 +89,15 @@ PlatformHookSerialPortInitialize (
       return Status;
     }
 
+ #if FixedPcdGetBool (PcdUseUniversalPayloadSerialPort) == 1
+
     Status = PcdSet32S (PcdSerialBaudRate, SerialPortInfo->BaudRate);

```

Comment:
Why does this PCD only affect BaudRate PCD?

Reply:
ARM utilizes PL011UartLib, PcdSerialBaudRate is fixed, and only PcdSerialBaudRate is defined fixed.
https://github.com/tianocore/edk2/blob/master/ArmPlatformPkg/Library/PL011UartLib/PL011UartLib.inf#L34